python_modules/dagster/dagster/_daemon/auto_run_reexecution/auto_run_reexecution.py

```diff
@@ -119,7 +120,17 @@ def retry_run(
         )
         return
 
-    external_pipeline = external_repo.get_full_external_job(failed_run.pipeline_name)
+    external_pipeline = code_location.get_external_pipeline(
+        PipelineSelector(
+            location_name=origin.code_location_origin.location_name,
+            repository_name=repo_name,
+            pipeline_name=failed_run.pipeline_name,
+            solid_selection=failed_run.solid_selection,
+            asset_selection=list(failed_run.asset_selection)
+            if failed_run.asset_selection
+            else None,
+        )
```

Comment:
is this one of those things where disaster strikes and it does everything instead of nothing if you pass in None instead of an empty list